src/olympia/editors/tests/test_helpers.py

```diff
@@ -1146,27 +812,19 @@ def test_pending_to_super_review(self):
                 ('Mozilla Add-ons: Delicious Bookmarks 2.1.072 flagged for '
                  'Admin Review'))
 
-    def test_nominated_review_time_set(self):
-        for status in REVIEW_ADDON_STATUSES:
-            for process in ['process_sandbox', 'process_preliminary',
-                            'process_public']:
-                if (status == amo.STATUS_UNREVIEWED and
-                        process == 'process_public'):
-                    continue
-                self.version.update(reviewed=None)
-                self.setup_data(status)
-                getattr(self.helper.handler, process)()
-                assert self.version.reviewed, ('Reviewed for status %r, %s()'
-                                               % (status, process))
-
-    def test_preliminary_review_time_set(self):
-        for status in amo.UNDER_REVIEW_STATUSES:
-            for process in ['process_sandbox', 'process_preliminary']:
-                self.file.update(reviewed=None)
-                self.setup_data(status)
-                getattr(self.helper.handler, process)()
-                assert File.objects.get(pk=self.file.pk).reviewed, (
-                    'Reviewed for status %r, %s()' % (status, process))
+    def test_nominated_review_time_set_version(self):
+        for process in ['process_sandbox', 'process_public']:
+            self.version.update(reviewed=None)
+            self.setup_data(amo.STATUS_NOMINATED)
+            getattr(self.helper.handler, process)()
+            assert self.version.reviewed
+
+    def test_nominated_review_time_set_file(self):
+        for process in ['process_sandbox', 'process_public']:
```

Comment:
Another two cases where the list could or should be a tuple since it's not being mutated later on.